index.html

```diff
@@ -0,0 +1,116 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+
+  <body>
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+
+    <!--
+    Hello Camper!
+    For now, the test suite only works in Chrome!
+    Please read the README below in the JS Editor before beginning.
+    Feel free to delete this message once you have read it.
+    Good luck and Happy Coding!
+    - The freeCodeCamp Team
+    -->
+    <!-- <h1>Hello World!.</h1>
+    <p>Hello World, we are software developers!</p>
+   -->
+   <style media="screen">
+@media only screen and (max-width: 780px) {
+  #nav-bar{
+    display: grid;
+  }
+}
+header{
+  display: flex;
+}
+</style>
```

Comment:
Please, for best HTML and CSS practices, it is advised not to use inline styles. Keep your style definition in a separate CSS file. It makes your code more organized.

Reply:
- [ ] Very good. You could also leave some links to resources that the student can use to learn best practices 👍🏾